Animated Mage/Archer classes

tanjha opened the pull request:
Found Art for Mage/Archer classes
Created animations from art
Rigged animations to class prefabs

tanjha committed:
Animated Mage/Archer classes

Found Art for Mage/Archer classes
Created animations from art
Rigged animations to class prefabs

tanjha commented:
Only conflict is for a save file

NamesAreHard6600 committed:
Merge branch 'main' into class_animation

NamesAreHard6600 reviewed (changes requested):
The mage animation completely fails on my end, and the archer gets stuck in the attack animation. There was also a lot of meta data it was yelling at me about. Maybe some unsaved files or something?

tanjha committed:
Fixed Mage + Archer animations

Re-rigged mage animation due to deletion on Github.
Fixed issues in Archer animation

tanjha committed:
Character select

Changed character select screen to have images for Mage and Archer classes. Tweaked Archer description and added Mage description.

NamesAreHard6600 committed:
Merge branch 'main' into class_animation

NamesAreHard6600 reviewed (approved):
Did code review and testing in discord. 
Merging
